upload-api/package.json

```diff
@@ -23,16 +23,17 @@
     "@web3-storage/access": "^10.0.0",
     "@web3-storage/capabilities": "^3.2.0",
     "@web3-storage/upload-api": "^1.0.4",
+    "@web3-storage/w3infra-ucan-invocation": "*",
```

Comment:
What is this module?

Reply:
to get constants names https://github.com/web3-storage/w3infra/pull/156/files#diff-daf55f5d4c62701298874a5ec82e56f7a9622d1bea49587e352c70a33cf77a6eR2